modules/learn/pages/data/validity.adoc

```diff
@@ -0,0 +1,70 @@
+= Validity Guarantees (ACID)
+
+[abstract]
+Couchbase Server validates data by means of _Atomicity_, _Consistency_, _Isolation_, and _Durability_.
+
+[#understanding-acid]
+== Understanding ACID
+
+Couchbase Server guarantees the validity of data that it handles.
+The properties whereby this is achieved are conventionally referred to by the acronym _ACID_; which stands for _Atomicity_, _Consistency_, _Isolation_, and _Durability_.
+
+The meanings of these properties, and the ways they are achieved by Couchbase Server, are described below.
+Note that various references are made to https://jepsen.io[Jepsen] &#8212; a project dedicated to improving the safety of distributed databases and other systems, with reference to specific https://jepsen.io/consistency[Consistency Models].
+
+[#atomicity-definition-and-implementation]
+=== Atomicity
+
+_Atomicity_ guarantees that when multiple data-elements need to be modified such that only the successful modification of all justifies the modification of any, _either_ all the modifications do occur successfully; _or_ none of them occurs.
+
+Atomicity is innately supported by the xref:learn:data/document-data-model.adoc[Couchbase Data Model]; which is naturally expressive of hierarchical data, and therefore typically only requires a single document for data that, under a relational model, would require multiple tables; and would therefore maintain data-elements discretely.
```

Comment:
So is this doc kinda explaining how ACID relates to both single doc operations, and multi-doc transactions?  I'm a touch confused by why this content is moved out of the transactions docs tbh - is it just to split a too-large doc?

Reply:
Well, I think if we do decide to define ACID, we need to do it for Couchbase Server and all its technologies. I think it will be perceived as odd if we describe some aspects, but not others. 

At any rate, the acronym includes Durability of course, and that is a separate page. So my sense is that the page logically needs to go on top, then transactions and durability beneath it.

Reply:
Note also that atomicity is something we already ascribe to the standard single-document model, which is pre-transaction.